docs/technical-setup/install-a-git-client.md

```diff
@@ -51,17 +97,14 @@ For this, launch a Terminal window, enter the command shown below, and follow th
 xcode-select --install
 ```
 
-<!--
-### Install Git on Windows
+### Install Git on Windows in WSL2
 
-We recommend installing the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) with the default Ubuntu Linux distribution.
-You can then install Git into your Ubuntu system:
+Once you have [installed WSL2](./install-wsl2.md), you can install Git using `apt`:
 
-```
-# You may need to update the package list first:
-# apt-get update
+```sh
+# First make sure apt is up to date
+sudo apt update
 
```

Comment:
Again, if this has been done recently (which it will have), we can skip this step